src/prism.c

```diff
@@ -2203,14 +2203,10 @@ pm_index_target_node_create(pm_parser_t *parser, pm_call_node_t *target) {
     pm_index_target_node_t *node = PM_ALLOC_NODE(parser, pm_index_target_node_t);
     pm_node_flags_t flags = target->base.flags;
 
-    if (parser->version != PM_OPTIONS_VERSION_CRUBY_3_3_0) {
```

Comment:
I know this gating was removed but I wonder if we want inequality for this by default with specific equality when it is locked to something specific.

`parser->version >= PR_OPTIONS_VERSION_CRUBY_3_3`

It is pretty likely we will see ranges doing `<=` and `>=` if something appears then is removed later.  Just a drive-by comment.

Reply:
That makes sense - I'll keep that in mind going forward.